lib/install/template.rb

```diff
@@ -8,7 +8,7 @@
   say "The JavaScript app source directory already exists"
```

Comment:
```diff
-   say "The JavaScript app source directory already exists"
+   say "The packs app source directory already exists"
```

Reply:
Thanks. updated via e91a315, also updated the `Javascript` to `packs` in line 10